packages/curve-common/src/widgets/Header/DesktopHeader.tsx

```diff
@@ -31,6 +34,8 @@ export const DesktopHeader = <TChainId extends number>({
   return (
     <>
       <AppBar color="transparent" ref={mainNavRef}>
+        <GlobalBanner {...BannerProps} />
```

Comment:
What's your opinion on using slots for this instead of tightly coupling `Banner.tsx` to `DesktopHeader.tsx`? Instead of passing `BannerProps` and importing, coupling and using a different component we could just have a `ReactNode` as use it directly. The user would then be able to pass whatever he wants as a banner component and whatever props are necessary.

Reference: https://www.dhiwise.com/post/mastering-react-slot-patterns-for-flexible-ui-components

Reply:
yeah it's also a good idea, but cannot be combined with the previous comment. This prop isn't adding much

Reply:
How about we convert the ChainProps, WalletProps and BannerProps to slots?

Reply:
Oh too bad, that doesn't work in this case :thinking: 
https://github.com/curvefi/curve-frontend/blob/fcef114d408b568e3980fe78d56370f1073f47e6/packages/curve-common/src/widgets/Header/MobileHeader.tsx#L112